device-types/Fortinet/FG-60E-POE.yaml

```diff
@@ -0,0 +1,44 @@
+---
+manufacturer: Fortinet
+model: FortiGate 60E-POE
+slug: fortinet-fg-60e-poe
+part_number: FG-60E-POE
+description: 8-Port Firewall Appliance with POE
+comments: '[Product Description](https://www.allfirewalls.de/en/Brands/Fortinet/FortiGate-Firewalls/Entry-Level-Firewalls/FG-60E-POE-Fortinet-FortiGate-60E-POE-Firewall.html)'
+u_height: 1
+is_full_depth: false
+front_image: false
+rear_image: false
+airflow: passive
+weight: 0.9
+weight_unit: kg
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
+  - name: PS1
+    type: dc-terminal
+    maximum_draw: 14
+    allocated_draw: 12
+    label: 12V DC
+interfaces:
+  - name: dmz
+    type: 1000base-t
+  - name: internal1
+    type: 1000base-t
```

Comment:
POE+ interfaces should have the following lines
`poe_mode: pse`
`poe_type: type2-ieee802.3at`